modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/filename/NodeFileTree.java

```diff
@@ -246,6 +244,9 @@ public class NodeFileTree extends SharedFileTree {
     /** Prefix for {@link #cacheStorage(CacheConfiguration)} directory in case of cache group. */
     private static final String CACHE_GRP_DIR_PREFIX = "cacheGroup-";
 
+    /** WAL state file name. */
+    private static final String WAL_STATE_FILE_NAME = "cdc-wal-state" + FILE_SUFFIX;
```

Comment:
FILE_SUFFIX must be made package private.

Reply:
CdcUtilityActiveCdcManager:

```
                File cdcModeFile = Paths.get(
                   cctx.kernalContext().pdsFolderResolver().fileTree().walCdc().getAbsolutePath(),
                    STATE_DIR,
                    CDC_MODE_FILE_NAME).toFile();

```

Path to the file must be calculated in `NodeFileTree`.